src/app/ngrx/effects/routing.effects.ts

```diff
@@ -196,11 +203,25 @@ export class RoutingEffects {
       params['episodes'] = combinedRouterState.episodeIds.join(',');
     }
 
-    this.router.navigate([
-      combinedRouterState.podcastSeriesId,
-      combinedRouterState.metricsType,
-      combinedRouterState.chartType,
-      combinedRouterState.interval.key,
-      params]);
+    switch (combinedRouterState.metricsType) {
+      case METRICSTYPE_DOWNLOADS:
```

Comment:
Is this to keep the chartType and interval out of the URL when they're the default?  Not entirely sure I follow.

Reply:
yes, exactly that. On prod right now, a page refresh on demographics or traffic sources will through a no routes match error because it includes the chart type and interval, segments that don't exist in their routes. Doh.